app/src/main/res/layout/partial_country_info.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="1dp"
+    android:background="@color/davy_grey"
+    android:foreground="?selectableItemBackground"
+    android:minHeight="150dp"
+    android:padding="@dimen/space_x2"
+    app:layout_goneMarginBottom="@dimen/space_half">
+
+
+    <TextView
+        android:id="@+id/txt_location"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/white"
+        android:textSize="14sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Indonesia" />
+
+    <TextView
+        android:id="@+id/txt_data"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="8dp"
+        android:gravity="center_vertical"
+        android:orientation="horizontal"
+        android:textAppearance="@style/TextAppearance.App.TextView.H5.White.Bold"
+        android:textColor="@color/color_confirmed"
+        android:textSize="24sp"
+        android:visibility="visible"
+        app:layout_constraintBottom_toTopOf="@+id/txt_data2"
+        app:layout_constraintEnd_toEndOf="@+id/txt_data2"
+        app:layout_constraintStart_toStartOf="@+id/txt_data2"
+        tools:text="200"
+        tools:visibility="visible">
+
+    </TextView>
```

Comment:
`TextView` isn't a view group, please close it with single tag `<TextView ... />`

Reply:
ops didnt notice that, thanks